app/controllers/api/destinations_controller.rb

```diff
@@ -8,6 +8,11 @@ def new
 
     def create
       destination.change
+      session[:undo] = {
+        action: 'undo',
+        undoable_action: 'change_next_page',
```

Comment:
i read `undoable` as something that cannot be undone 😆

Reply:
will keep until we find a better name